docs/backup-recovery.md

```diff
@@ -0,0 +1,38 @@
+# Backup and Recovery
+
```

Comment:
Add a note stating the backup plugin was removed in 0.5.x as it was completely broken or something.